### ISSUE TITLE
Cannot find accessibilityLabel in NavigationBar

### ISSUE THREAD

**Issue by cybertk** [kif-framework/KIF#422]
KIF time out when calling waitForViewWithAccessibilityLabel() for BarButtonItem in NavigationBar
